deepsubtree.go

```diff
@@ -16,23 +17,134 @@ type DeepSubTree struct {
 	*MutableTree
 }
 
+type DSTNonExistenceProof struct {
```

Comment:
All exports should have comments, not just Functions and Methods, but Types as well.